README.md

```diff
@@ -1 +1,22 @@
-# DSCI-532_2024_9_solar-savers
\ No newline at end of file
+# Solar Savers 
+
+A dashboard application helping Canadian homeowners in solar panel purchasing decisions.
+
+## Purpose  
+
+With escalating inflation and growing environmental consciousness, consumers across Canada seek to save money where they can. Solar Savers, a user-friendly app, is designed to better equip homeowners in the decision-making process of purchasing solar panels for their home. With user's location, electricity rates, and panel quality, Solar Savers allows users to see the potential electricity and financial savings, helping them decide whether investing in solar panels is worthwhile.
+
+## Contributing
+
+Interested in contributing to this project? Check out the [contributing guidelines](CONTRIBUTING.md). *Please note: This project is released with a Code of Conduct. By contributing to this project, you agree to abide by the terms in the [Code of Conduct](CODE_OF_CONDUCT.md).* 
+
+## License
+
+Solar Savers is developed and maintained by Marco Bravo, Nicole Bidwell, Rafe Chang, and Sam Fo. The code is licensed under the MIT license. Please see `LICENSE.md` for details.  
```

Comment:
Overall, LGTM. However, should we also mention the Open Canada and Creative Commons Licenses here? (Although the pull request containing such change has not been approved yet).

Reply:
```suggestion
Solar Savers is developed and maintained by Marco Bravo, Nicole Bidwell, Rafe Chang, and Sam Fo. The code is licensed under the MIT License, Open Canada License, and Creative Commons License. Please see `LICENSE.md` for details.  
```